### ISSUE TITLE
Suggestion: use gavanderhoorn/rxcpp_vendor

### ISSUE THREAD

**Issue by gavanderhoorn** [rosin-project/rxros#12]
I've just created a Catkin package out of RxCpp: [gavanderhoorn/rxcpp_vendor](https://github.com/gavanderhoorn/rxcpp_vendor).

This allows us to package RxCpp irrespective of what upstream is doing (they don't provide Debian packages) and we don't need to ship copies of the headers ourselves (currently in [rxros/include/rxcpp](https://github.com/rosin-project/rx_ros/tree/467f6c8b47f087b8b8b97224033cf6be97e52f3e/rxros/include/rxcpp)), which increases maintenance effort and has other disadvantages.

To use it, add the following to the package manifest of `rxros`:

```xml
  <build_depend>rxcpp_vendor</build_depend>
  <build_export_depend>rxcpp_vendor</build_export_depend>
```

And in the `CMakeLists.txt` add `rxcpp_vendor` both to the `find_package(catkin REQUIRED COMPONENTS ..)` and the `catkin_package(CATKIN_DEPENDS .. )` lists.

And of course make sure to have a clone of `gavanderhoorn/rxcpp_vendor` in the workspace.

**Comment by gavanderhoorn** [comment]
Implemented in #14.

**Comment by gavanderhoorn** [comment]
Fixed in #14 and rosin-project/rxros_examples#1.
